Vecherskiys/02/Task2.2v1.html

```diff
@@ -1,10 +1,11 @@
 <!DOCTYPE html>
 <HTML>
  <HEAD>
+ <meta name="Description" content="Weather">
```

Comment:
meta tag for charset https://www.w3schools.com/tags/att_meta_charset.asp

Reply:
Ok